windows/README.md

```diff
@@ -5,7 +5,7 @@ This project provides a smoother way to install and run Kolibri in a Windows Mac
 
 ---
 #### This project was built using the following software:
-* Windows 8.1 32 bit.
+* Windows 8.1 and above.
```

Comment:
The headline is `This project was built using the following software` - so I think it's also important to be specific about which Windows should be used for building it.

If the installer can be built on other platforms than Windows 8.1 32 bit, and it's tested, then let's also list those (specifically). For instance "Building is known to work on: Windows 8.1 32 bit". And then in addition to that, you could write that "Windows 8.0 and below" are incompatible?